src/framework/Framework_Snapshot_001.test.js

```diff
@@ -0,0 +1,46 @@
+import {
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+  screenshot,
+  webKitBrowserOps,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'Framework snapshot test',
+  description: 'Tests if the Framework snapshot module works',
+  teardown: restartFramework,
+  steps: [
+    {
+      description: 'Deactivating Snapshot Plugin',
+      test: pluginDeactivate,
+      params: constants.snapshotPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Activating Snapshot Plugin',
+      test: pluginActivate,
+      params: constants.snapshotPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Deactivating and Activating WebKitBrowser plugin',
+      test: webKitBrowserOps,
```

Comment:
Same as others, probably good to rename.

Reply:
Rename this to webKitBrowserStartAndResume